### PR TITLE
Create usage guide and sample code for developers

### PR DESCRIPTION
**Pull request by jleach**
This PR addresses #185 by providing a usage guide and sample policy for developers.

### CODE REVIEW

**Comment by jleach** [comment]
@stewartshea @mitovskaol This is a WIP. Just pushing out the first few lines so anyone who is interested can follow along. I'll remove the WIP label when its ready go be reviewed.

**Comment by WadeBarnes** [comment]
@jleach, Looking good so far.  Feel free to use the working FPO templates as examples if you like.

File: security/aporeto/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+**🤓 ProTip**
+
+* K.I.S - Keep It (your labels) Simple; don't try and outsmart yourself.
+* Where possible use the naming convention proposed here for uniformity across projects.
+
+## Policy
+
+The policy below will provide most teams enough to get up and running in short order; Customize them with labels or template parameters as needed. If you find the sample policy below does not suite your needs contact platform services to help create more advanced policy.
+
+In the subsections below we'll use one or more tags to identify the source and destination systems. The relevance if `source` and `destination` is that the application identified in the `source` will be able to open a network connection to the application identified in `destination`; Once a connection is open then data is able to flow bidirectionally.
```
Comment (mitovskaol) [inline security/aporeto/docs/README.md]: If the destination is protected by Aporeto as well, then a matching custom policy on that end is required in order to make the flow bidirectional.

File: security/aporeto/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+With the addition of new security components Platform Services is able to offer product teams the ability to operate in a Zero Trust model; In a Zero Trust is a security model you don't trust anything outside of your own components (pods).
+
+This is done by way of creating application identities for each component (Web, API, Database, etc) and based on this identity allowing specific components to talk to one another by creating `NetworkSecurityPolicy` (NSP) or `ExternalNetworks` (EN).
+
+**🤓 ProTip**
+
+* Assume that the network and platform are insecure and built up robust security practices.
+
+## How it Works
```
Comment (mitovskaol) [inline security/aporeto/docs/README.md]: Would be great to have "Step 1", "Step 2", etc added to the guide section titles.

File: security/aporeto/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+
+**Step 1 - Add Labels***
+
+Open your deployment manifest and add labels to your `DeploymentConfig` sections to uniquely identify each one:
+
+![Add Labels](add_labels.gif)
+
+In this illustration I have a minio deployment that I add the label `role: objstore` to and an API deployment I add `role: api` to.
+
+**Step 2 - Add Policy**
```
Comment (mitovskaol) [inline security/aporeto/docs/README.md]: Added a comment that if an app already has a base custom policy set up, they would have to remove it first, before adding the new policy.

File: security/aporeto/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+
+### Advanced
+
+## Projects
+
+This is a list of some projects that have already implemented a Zero Trust security model:
+
+[Family Protection Order](https://github.com/bcgov/Family-Protection-Order)
+
+### Troubleshooting
```
Comment (mitovskaol) [inline security/aporeto/docs/README.md]: Add Step 3 - Test it
This section can include a link to the TestConnection script https://github.com/BCDevOps/openshift-developer-tools/blob/master/bin/testConnection and examples on how to use it.

File: security/aporeto/docs/README.md (hunk 1 above)
Comment (jleach) [reply security/aporeto/docs/README.md]: Yes, if they are trying to talk across namespaces then policy is needed on both ends. Within the same namespace its not required.

File: security/aporeto/docs/README.md (hunk 3 above)
Comment (jleach) [reply security/aporeto/docs/README.md]: Added.

File: security/aporeto/docs/README.md (hunk 2 above)
Comment (jleach) [reply security/aporeto/docs/README.md]: I'll add a ToC so people can quickly navigate to different sections. I think I like it as is though.

File: security/aporeto/docs/README.md (hunk 4 above)
Comment (jleach) [reply security/aporeto/docs/README.md]: Added.

File: security/aporeto/docs/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+* Assume that the network and platform are insecure and built up robust security practices.
+
+## Table of Contents
+
+[Quick Start](./QuickStart.md)
+
+[Custom Network Security Policy](./CustomPolicy.md)
+
+[Custom External Networks](./CustomNetworks.md)
```
Comment (stewartshea) [inline security/aporeto/docs/README.md]: this points to the wrong file even though the file is under construction

File: security/aporeto/docs/README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+| Channel         | Description     |
+| --------------- |:----------------|
+| #devops-sos     | Use this channel when things are on fire 🔥 and you need immediate help to resolve a production problem. |
+| #devops-how-to  | Use this channel to tap into the top-notch OCP community for help. |
+
+## Projects
+
+This is a list of some projects that have already implemented a Zero Trust security model:
+
+[Family Protection Order](https://github.com/bcgov/Family-Protection-Order)
```
Comment (stewartshea) [inline security/aporeto/docs/README.md]: is it possible to add a link to where their example code is, or do we just say to reach out to this team if they want to see the "living" code

**Pull request merged**